release.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+
+# Usage
+# PUBLISH_GEM=true RELEASE_VERSION=1.2.7 ./release.sh
+
+PHRASEAPP_RUBY_REPO=git@github.com:phrase/phraseapp-ruby.git
+PHRASEAPP_RUBY_TMP=$(mktemp -d /tmp/phraseapp-ruby-lib-XXXX)
```

Comment:
I would probably try to check if that works `if [[ -z $PHRASE_RUBY_TMP]]; then echo "unable to create tmp dir"; exit 1; fi` as there could be an issue here (inodes, permissions, etc...)